src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ *  An interface to produce a printable string representation.
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Good job in creating the interface. Based on the LO class diagram, `getPrintableString()` should have the `public` keyword. However if you do not, all methods in an interface are public, even if you leave out the public keyword in the method declaration.